{"name": "test_al[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 != 2", "trace": "self = <fff.Test_allure object at 0x0000027092DF1C18>, a = 2\n\n    @pytest.mark.parametrize(\"a\",[1,2,3,4,5])\n    @allure.severity(allure.severity_level.BLOCKER)\n    \n    @allure.step('我是测试步骤001')\n    def test_al(self, a):\n>       assert a != 2\nE       assert 2 != 2\n\nscripts\\fff.py:12: AssertionError"}, "steps": [{"name": "我是测试步骤001", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 != 2\n", "trace": "  File \"d:\\program files\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 157, in impl\n    return func(*a, **kw)\n  File \"D:\\python\\Rep\\scripts\\fff.py\", line 12, in test_al\n    assert a != 2\n"}, "parameters": [{"name": "a", "value": "2"}], "start": 1593882610463, "stop": 1593882610463}], "parameters": [{"name": "a", "value": "2"}], "start": 1593882610463, "stop": 1593882610464, "uuid": "19e42e21-af46-4840-9368-e91ea3962f3b", "historyId": "2b294d70de3044a60588d9e389013dd3", "testCaseId": "879567248dbe0c1e640f1107f4b35cdd", "fullName": "scripts.fff.Test_allure#test_al", "labels": [{"name": "severity", "value": "blocker"}, {"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "fff"}, {"name": "subSuite", "value": "Test_allure"}, {"name": "host", "value": "bbr1914"}, {"name": "thread", "value": "9628-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.fff"}]}